codecept.conf.js

```diff
@@ -0,0 +1,24 @@
+const { setHeadlessWhen, setCommonPlugins } = require('@codeceptjs/configure');
+// turn on headless mode when running with HEADLESS=true environment variable
+// export HEADLESS=true && npx codeceptjs run
+setHeadlessWhen(process.env.HEADLESS);
+
+// enable all common plugins https://github.com/codeceptjs/configure#setcommonplugins
+setCommonPlugins();
+
+/** @type {CodeceptJS.MainConfig} */
+exports.config = {
+  tests: './*_test.js',
```

Comment:
```suggestion
  tests: './src/tests/e2e/**/*.spec.ts',
```
경로가 이렇게 되어야할 것 같아요

spec으로 할지, test로 할지는 추가로 논의되면 좋을 것 같습니당